examples/jsm/loaders/LDrawLoader.js

```diff
@@ -1903,11 +1903,11 @@ class LDrawLoader extends Loader {
 		this.partsLibraryPath = '';
 
 		// Material assigned to not available colors for meshes and edges
-		this.missingColorMaterial = new MeshStandardMaterial( { color: 0xFF00FF, roughness: 0.3, metalness: 0 } );
+		this.missingColorMaterial = new MeshStandardMaterial( { name: Loader.DEFAULT_MATERIAL_NAME, color: 0xFF00FF, roughness: 0.3, metalness: 0 } );
```

Comment:
The names are overwritten below.

Reply:
Good catch, thanks. Fixed: d2d445b52c9b840cf52297c8077691aa43241f0b